media-sound/mp3diags/mp3diags-1.5.03.ebuild

```diff
@@ -21,28 +21,18 @@ DEPEND="
 	dev-libs/boost:=
 	dev-qt/qtcore:5
 	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
```

Comment:
What's the story with this (the old ebuild used to sed out the dep)?

Reply:
(And please put whatever that story is in the commit message if not retaining that)

Reply:
the resulting binary links against it

Reply:
Thanks. For future reference, for these things, I really like to link to the relevant bug (the old one, with `Bug:`) and make some remark about it in the commit message to make clear it's not a mistake.